### PR TITLE
Add DeepOnion

### PR DESCRIPTION
**Pull request by nezero**
## Description

Add the DeepOnion Blockchain to the project

## Testing instructions

Unit tests as per the integration guide have been added.

## Types of changes

Added DeepOnion coin and blockchain info to the project as well as integration tests.

- [X] Prefix PR title with `[WIP]` if necessary.
- [X] Add tests to cover changes as needed.
- [X] Update documentation as needed.

### CODE REVIEW

**Comment by Pattamapor** [comment]
Yes

**Comment by vikmeup** [comment]
@nezero can you take a look at the requirements before moving any further: https://developer.trustwallet.com/wallet-core/newblockchain#requirements

**Comment by nezero** [comment]
> @nezero can you take a look at the requirements before moving any further: https://developer.trustwallet.com/wallet-core/newblockchain#requirements
> 
> > **[Adding Support for a New Blockchain](https://developer.trustwallet.com/wallet-core/newblockchain#requirements)**

Yes, I read this before starting. The only point that we don't meet is "top 100 on CMC" and this is not something we can't really control unless we follow some of the other projects and generate a few trillion coins, though fake market caps is not what our project is about.

File: coins.json [hunk 1]
```diff
@@ -804,6 +804,25 @@
         "url": "https://aeternity.com",
         "rpcNodeInfo": "https://github.com/aeternity/aeternity"
     },
+    {
+        "id": "deeponion",
+        "name": "DeepOnion",
+        "symbol": "ONION",
+        "decimals": 8,
+        "blockchain": "DeepOnion",
```
Comment (hewigovens) [inline coins.json]: it's a bitcoin fork?

**Comment by nezero** [comment]
@hewigovens had an email that you commented, but I can't see it.

It is a bitcoin fork, however the blockchain is modified slightly with the addition of a timestamp on transactions.

**Comment by nezero** [comment]
Can someone trigger the iOS Tests again?

Looks like there was an infrastructure issue while the tests were running.

`Job issues
·
1 error
The agent: Hosted macOS 2 lost communication with the server. Verify the machine is running and has a healthy network connection. For more information, see: https://go.microsoft.com/fwlink/?linkid=846610`

**Comment by nezero** [comment]
```
```

**Comment by nezero** [comment]
@hewigovens can anyone help us out with a review on this?

**Comment by hewigovens** [comment]
Thanks for your efforts, but as https://github.com/trustwallet/wallet-core/pull/528#issuecomment-504722853 said, let's close it now

**Pull request closed without merging**